### PR TITLE
merge 1

### PR DESCRIPTION
**Pull request by Genya9000**
merge 1

### CODE REVIEW

**Comment by EvgenKhaliman** [comment]
Круто!

File: .idea/misc.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       </list>
```
Comment (valeriiafatova) [inline .idea/misc.xml]: Should not be present in repository

File: src/main/java/servises/Controller.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package servises;
+
+import module.Banks;
+import module.Model;
+
+public class Controller extends Model {
+    public Banks goodadvise(){
+        if((getPurchase().equals("машина"))&&(getEarlyRepayment().equals("да"))&&(getCreditLineIncrease().equals("да"))||(getPurchase().equals("квартира"))&&(getEarlyRepayment().equals("нет"))&&(getCreditLineIncrease().equals("нет"))) return Banks.SWED;
```
Comment (valeriiafatova) [inline src/main/java/servises/Controller.java]: Do not use Cyrillic alphabet

File: src/main/java/servises/Controller.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        if((getPurchase().equals("машина"))&&(getEarlyRepayment().equals("нет"))&&(getCreditLineIncrease().equals("да"))||(getPurchase().equals("квартира"))&&(getEarlyRepayment().equals("да"))&&(getCreditLineIncrease().equals("нет"))) return Banks.PRIVAT;
```
Comment (valeriiafatova) [inline src/main/java/servises/Controller.java]: Make Formatting

File: src/main/java/servises/Main.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package servises;
+
+import view.Creditline;
+
+public class Main {
+    public static void main(String[] args) {
+          new Creditline();
+          /*Model model = new Model();
```
Comment (valeriiafatova) [inline src/main/java/servises/Main.java]: remove commented code

File: src/main/java/servises/Main.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+          System.out.println(model.getPurchase());*/
+          Controller controller = new Controller();
+          System.out.println("Ваши условия выполняются только в банке " + controller.goodadvise());
```
Comment (valeriiafatova) [inline src/main/java/servises/Main.java]: Do not use Russian in code at all

File: src/main/java/servises/Main.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+package servises;
+
+import view.Creditline;
+
+public class Main {
+    public static void main(String[] args) {
+          new Creditline();
+          /*Model model = new Model();
+          System.out.println(model.getPurchase());*/
+          Controller controller = new Controller();
```
Comment (valeriiafatova) [inline src/main/java/servises/Main.java]: You only create object. But do not call method to run programm

File: src/main/java/servises/Controller.java (hunk 2 above)
Comment (valeriiafatova) [reply src/main/java/servises/Controller.java]: Refactor method to not contain more then 3 condition

File: src/main/java/view/Creditline.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package view;
+
+import java.io.IOException;
+
+public class Creditline extends Repayment {
+    public Creditline() {
+        System.out.println("Вас интересует возможность увеличения кредитной линии ?(да/нет)");
```
Comment (valeriiafatova) [inline src/main/java/view/Creditline.java]: NO LOGIC in constructor

File: src/test/java/servises/ControllerTest.java [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    @Test
+    public void returnIboxtoo() {
+        controller.setPurchase("квартира") ;
+        controller.setEarlyRepayment("да");
+        controller.setCreditLineIncrease("да");
+
+        Banks result = controller.goodadvise();
+        assertEquals(Banks.IBOX, result);
+    }
+}
```
Comment (valeriiafatova) [inline src/test/java/servises/ControllerTest.java]: .class should not be present in repo!!!!

**Pull request merged**